ashpd-demo/src/widgets/gst_paintable.rs

```diff
@@ -96,6 +96,23 @@ impl CameraPaintable {
             .unwrap();
         let paintable = sink.property::<gdk::Paintable>("paintable");
 
+        let (convert, sink) = if paintable
```

Comment:
Is there a reason to define convert here in the tuple? can't the definition be placed before the `let sink = {} ` block?

Reply:
leftover, if pw was giving us a gl/dmabuf buffer we would be using `glcolorconver` instread of `videoconvert`